RELEASE_NOTES.md

```diff
@@ -26,6 +26,7 @@
   - NaN values are treated as missing when gaps are determined in the `OrderedRingBuffer`.
   - Provide access to `capacity` (maximum number of elements) in `MovingWindow`.
   - Methods to retrieve oldest and newest timestamp of valid samples are added to both.
+  - Len Magic methods are replaced by `valid_size` property in both.
```

Comment:
Maybe add that `len(window)` should be replaced with `window.valid_size()` more explicitly?